docs/platforms/javascript/common/feature-flags/index.mdx

```diff
@@ -0,0 +1,45 @@
+---
+title: Set Up Feature Flags
+sidebar_order: 7000
+notSupported:
+  - javascript.aws-lambda
+  - javascript.azure-functions
+  - javascript.bun
+  - javascript.capacitor
+  - javascript.cloudflare
+  - javascript.connect
+  - javascript.cordova
+  - javascript.deno
+  - javascript.electron
+  - javascript.express
+  - javascript.fastify
+  - javascript.gcp-functions
+  - javascript.hapi
+  - javascript.koa
+  - javascript.nestjs
+  - javascript.nodejs
+  - javascript.wasm
+description: "Learn how to set up feature flag evaluation tracking and feature flag change tracking."
+---
+
+<Alert level="info" title="Currently in Beta">
+
+The support for **feature flag change tracking** and **feature flag evaluation tracking** is currently in beta.
```

Comment:
```suggestion
The support for **feature flag change tracking** and **feature flag evaluation tracking** is currently in closed beta. If you'd like to be added to the beta, please fill out [this form](https://forms.gle/EeNwTepvVwt7poAJ8).
```

Reply:
@jas-kas They'll see the setup fly out in the Sentry UI and the feature flags UI.